pom.xml

```diff
@@ -97,6 +97,11 @@
       <artifactId>org.everit.json.schema</artifactId>
       <version>1.11.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.neovisionaries</groupId>
+      <artifactId>nv-i18n</artifactId>
+      <version>1.27</version>
+    </dependency>
```

Comment:
What is this dependency? Why has it been introduced?

Reply:
It is a library for converting contry codes into country names, 
It contains data similar to the one that is used in folio - https://github.com/folio-org/stripes-smart-components/blob/e104573a2ee448fdb520ea5bedf3b3f08cd5e176/lib/AddressFieldGroup/data/countries.js

Since such convertion uses ISO standards, we can use library instead of hardcoding such data inside of module